src/client/al-api-client.ts

```diff
@@ -718,7 +718,11 @@ export class AlApiClient implements AlValidationSchemaProvider
       let response = await this.axiosRequest( endpointsRequest );
       Object.entries( response.data ).forEach( ( [ serviceName, endpointHost ] ) => {
           let host = endpointHost as string;
-          host = host.startsWith("http") ? host : `https://${host}`;      //  ensuring domains are prefixed with protocol
+          if (host.startsWith("async")) { // naming convention for WebSocket services
+            host = `wss://${host}`; // add prefix for websocket protocol
+          } else {
+            host = host.startsWith("http") ? host : `https://${host}`;      //  ensuring domains are prefixed with protocol
```

Comment:
This is actually an `else if` disguised as an `else` + ternary.  Avoiding unnecessary reassignment can avoid accidents, so I'd prefer

```
else if ( host.startsWith( "http" ) {
    host = `https://${host}`;
}